content/en/docs/concepts/configuration/manage-resources-containers.md

```diff
@@ -237,6 +238,73 @@ are available in your cluster, then Pod resource usage can be retrieved either
+when a process uses memory as a work area and when using memory-backed
+`emptyDir`. But when using memory as a volume like memory-backed `emptyDir`,
+there are additional points below that you should be careful.
+
+* Files stored on a memory-backed volume are almost entirely managed by the
+  user application. Unlike when used as a work area for a process, you can not
+  rely on like garbage collection by the runtime for language.
+* The purpose of writing files to a volume is to save data or pass it between
+  applications. Deletion of files may not be considered or may depend on other
+  applications, then tend to persist for long time as files accumulate or are
+  not deleted as intended.
+* The memory-backed `emptyDir` is useful when writing large amounts of logs
+  continuously or passing large image files because of its performance.
+  However, memory is very small compared to disks, and using a large amount of
+  memory will affect the normal operation of the node, so you must be very
+  careful to use the minimum amount necessary.
+
+If you use memory backed `emptyDir` without appropriate settings, e.g. without
+`sizeLimit` nor memory limits, the following risks may occur:
+
+* A single Pod with memory backed `emptyDir` may unintentionally occupy node's memory.
+* If multiple memory backed `emptyDir`s are created on the same mode, their total
+  logical size can easily exceed the amount of memory implemented on the node, and
+  they can be consumed very quickly due to its performance.  This results in memory
+  exhaustion due to the unintentional or malicious creation of a large number of
+  memory backed `emptyDir`s.
+* As usage of memory backed `emptyDir` increases, Linux will eventually try to
+  OOMkill this container, but if a large number of such `emptyDir`s are created
+  and consumed rapidly, the tmpfs volume may be not deleted before the node run
+  out of memory and crash.
+
+To prevent these risks, everyone who can define a Pod should specify memory
+limits for pod or `sizeLimit` for `emptyDir`.
+
+Since memory limits apply to all pod memory usage, and the `sizeLimit` only
+applies to the `emptyDir`, a memory limits is the proper tool to prevent a
+container from consuming too much of the node's allocatable memory.
+The `sizeLimit` is really a tool to prevent usage of the memory-backed `emptyDir`
+from causing the container to hit its memory limit and OOM, not a tool to limit
+a pod's memory uasge. So it is better concidering to specify memory limits for
+pod first.
+
+If you are administering a cluster or namespace, you also set
+[ResourceQuota](/docs/concepts/policy/resource-quotas/) that limits memory use;
+you may also want to define a [LimitRange](/docs/concepts/policy/limit-range/)
+for additional enforcement.
+If you specify a `spec.containers[].resources.limits.memory` for each Pod,
+then the default size of an `emptyDir` volume will be `.spec.resources.limits.memory`.
```

Comment:
change "default size" to "maximum size" ?

Reply:
Change "`.spec.resources.limits.memory`" to "the pod's memory limit"   (you already spelled out the field path before)